setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E731,E266,E501,C901,W503,E203
+ignore = E731,E266,E501,C901,W503,E203,E704
```

Comment:
As noted [here](https://github.com/psf/black/issues/3887), flake8's E704 conflict's with the new concise dummy implementation style